timeline.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Timeline</title>
+
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="portfoliostyle.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	
+</head>
+<body>
+	<!-- navbar -->
+	<nav>
+		<a href="portfolio.html" class="nav-item">Home</a>
+		<a href="eleanor-roberts-cv.pdf" class="nav-item cv" target="_blank">CV</a>
+	</nav>
+	<!-- header -->
+	<header id="home">
+		<h1>History of Me</h1>
+	</header>
+	<!-- timeline -->
+	<section id="timeline">
+		<div class="timeline">
+			<!-- left container -->
+			<div class="container left">
+				<div class="content">
+					<h3>Risk Management Operations Specialist</h3>
+					<p class="timeline-company">Celgene/BMS
+					</p>
+					<p class="timeline-date">May 2019 - October 2020
+					</p>
+					<p class="to-hide">In my role as (Data Entry) RMO Specialist, I did many things with my time which were all very impressive and involved lots of PV which I absolutely love and was not the reason I then transferred out of the industry and into coding which is much more suied to my tastes. I love logic puzzles, and whilst the overly large excel spreadsheets at Celgene/BMS were satisfying to correctly create/adjust/filter they are not as fun as what is essentially a big beautiful new rubix cube for me to play with.
+					</p>
+				</div>
+			</div>
+			<!-- right container -->
+			<div class="container right">
+				<div class="content">
+					<h3>Bartending</h3>
+					<p class="timeline-company">The Lord Nelson
+					</p>
+					<p class="timeline-date">September 2016 - May 2019
+					</p>
+					<p class="to-hide">Working as a bartender at this local pub, I had a great time making friends with the regulars and seeing them get spectacularly drunk (and occassionally helping them get home afterwards). I worked on my beautiful social skills, ability to let drunken bulls**t roll off my back, and general stock management/time management/interpersonal and management skill improvement.
```

Comment:
Text overflows box on iphone 5 at management/interpersonal section.

Reply:
Edited the text for less run along. Will have to revisit depending on actual content 🧐